src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -361,14 +362,58 @@ invalidated when a device disconnects. This means your code should always
 retrieve (through `getPrimaryService(s)`, `getCharacteristic(s)`, etc.) these
 attributes after reconnecting.
 
+### Read and write to Bluetooth Descriptors
+
+Bluetooth GATT descriptors are defined attributes that describe a
```

Comment:
Change 'defined attributes' to 'attributes'. If 'defined' is adding something I'm not aware of, then explain it.